stream.go

```diff
@@ -308,7 +311,7 @@ SEND_CLOSE:
 }
 
 // forceClose is used for when the session is exiting
-func (s *Stream) forceClose() {
```

Comment:
?

Reply:
The current semantics meant that when `sendMsg` faces a write timeout, the back pressure indicates that if the queue fills up. If we try for per-stream queues, or otherwise increase the overall queue size, while wanting to preserve those sematics, we'll need to track that the stream closed because of a write timeout in order to report that it's a timeout rather than a reset.

Reply:
But we're not actually using this error.